includes/class-sensei-course.php

```diff
@@ -145,17 +145,81 @@ public function __construct() {
 
 		add_action( 'template_redirect', [ $this, 'setup_single_course_page' ] );
 		add_action( 'sensei_loaded', [ $this, 'add_legacy_course_hooks' ] );
+
+		// Add custom navigation.
+		add_action( 'in_admin_header', [ $this, 'add_custom_navigation' ] );
+		add_action( 'parent_file', [ $this, 'highlight_menu_item' ] );
 	}
 
 	/**
-	 * Register and enqueue scripts that are needed in the backend.
+	 * Highlight the menu item for the course pages.
+	 *
+	 * @param string $file
```

Comment:
The `@since` is missing. :)

Reply:
Fixed here: https://github.com/Automattic/sensei/commit/e5f525110cc9d3e2e4a8dc26d9e3b36108d6a5b8